Model/Sync/Catalog/Processor/Main.php

```diff
@@ -689,7 +689,7 @@ public function removeProductFromSyncTable($productIds, $storeId)
      */
     public function isProductParentYotpoIdFound($yotpoData, $parentId): bool
     {
-        return isset($yotpoData[$parentId]) && isset($yotpoData[$parentId]['yotpo_id']);
+        return isset($yotpoData[$parentId]) && isset($yotpoData[$parentId]['yotpo_id']) && $yotpoData[$parentId]['yotpo_id'];
```

Comment:
is this not 0 then? shouldn't this be `&& **!**$yotpoData[$parentId]['yotpo_id'];`
but regardless, let's make this 
`&& $yotpoData[$parentId]['yotpo_id'] === 0;`
Or 
`&& $yotpoData[$parentId]['yotpo_id'] !== 0;`
for explicitness

Reply:
discussed offline.